src/front/ticketer/lib/pages/common/event_tile.dart

```diff
@@ -0,0 +1,135 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:ticketer/backend_communication/model/event.dart';
+
+class EventTile extends StatefulWidget {
+  final Event event;
+
+  const EventTile({Key? key, required this.event}) : super(key: key);
+
+  @override
+  State<EventTile> createState() => _EventTileState();
+}
+
+class _EventTileState extends State<EventTile> {
+  late Event _event;
+
+  Widget _eventContainer() {
+    return InkWell(
+      onTap: () => _showEventDetails(),
+      child: Container(
+        padding: const EdgeInsets.all(9.0),
+        decoration: BoxDecoration(border: Border.all(color: Colors.blueGrey)),
+        child: SingleChildScrollView(
+          child: Column(
+            children: [
+              _getEventInfo("Event name", _event.name,
+                  style: const TextStyle(fontWeight: FontWeight.bold)),
+              _getEventInfo("Date & time", _event.date),
+              _getEventInfo("Location", _event.location),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Future<void> _showEventDetails() async {
+    await showDialog(
+      context: context,
+      builder: (context) {
+        return AlertDialog(
+          title: Text(_event.name),
+          scrollable: true,
+          content: SingleChildScrollView(
+            child: SizedBox(
+              width: MediaQuery.of(context).size.width / 4,
+              height: MediaQuery.of(context).size.height / 2,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text(
+                    _event.description,
+                    textAlign: TextAlign.justify,
+                  ),
+                  const Text(
+                    "\nSectors: ",
+                    style: TextStyle(
+                      fontWeight: FontWeight.w600,
+                      color: Colors.blueAccent,
+                    ),
+                  ),
+                  SingleChildScrollView(
+                    child: ListView(
+                      shrinkWrap: true,
+                      children: _event.sectors
+                          .map((s) => Text(s.toString()))
+                          .toList(),
+                    ),
+                  )
+                ],
+              ),
+            ),
+          ),
+          actions: [
+            ElevatedButton(
+              onPressed: () => {
+                Navigator.pop(context),
+              },
+              child: const Text('OK'),
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  Widget _getEventInfo(String property, String value, {TextStyle? style}) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Flexible(
+          flex: 2,
+          child: Text(
+            property,
+            style: const TextStyle(
+                fontWeight: FontWeight.w700, color: Colors.blue, fontSize: 16),
```

Comment:
we shouldn't use outright colors. This is like magic numbers when You change color scheme